js/samples/05.chatModeration/src/index.ts

```diff
@@ -38,12 +33,16 @@ const botFrameworkAuthentication = new ConfigurationBotFrameworkAuthentication(
 // See https://aka.ms/about-bot-adapter to learn more about how bots work.
 const adapter = new CloudAdapter(botFrameworkAuthentication);
 
+// Create storage to use
+//const storage = new MemoryStorage();
+
```

Comment:
Should this be uncommented?

Reply:
I just removed it... that's done in the bot.ts file for that sample